schema/pet-insurance.json

```diff
@@ -592,5 +569,4 @@
             "description": "Breed type is required by the website"
         }
     ]
-
 }
```

Comment:
!

Reply:
That's not an inserted hyphen. It represent a deleted empty line.
![image](https://user-images.githubusercontent.com/34474056/54904051-11d30f00-4ee6-11e9-838e-9b1d9b0d5a7b.png)

Reply:
I undertand that :) Configure your editor to emit a single blank line when saving a file.

Reply:
Always end the lines with newline, misconfigured editor.